### PR TITLE
feat: [SRM-11790]: Implemention of new SLOs listing page

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. New SLO's listing page

#### Screenshots
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

## [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 47 Issues 
* Code issue 47 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11127)

**Comment by harshilbits** [comment]
retrigger titlecheck

**Review by devinder-harness — APPROVED** [review]

File: src/modules/85-cv/pages/slos/components/SLOCardSelect/SLOCardSelect.tsx [hunk 1]
```diff
@@ -6,25 +6,47 @@
...
   displayColor: string
 }
 
 const SLOCardSelect: React.FC<SLOCardSelectProps> = ({ displayName, count, displayColor }) => {
-  const dataTooltipId = useMemo(() => displayName?.replace(/ /g, '') + '_tooltip', [displayName])
+  const riskCategory = displayName?.toUpperCase()?.replace(/ /g, '_') as RiskTypes
+  const dataTooltipId = useMemo(() => riskCategory + '_tooltip', [displayName])
+  const iconCardBackgroundColor = getRiskColorValue(riskCategory)
+  const riskCategoryLogo = getRiskColorLogo(riskCategory) as IconName
+  // const isRiskCategoryHealthOrUnhealthy = riskCategory === RiskValues.HEALTHY || riskCategory === RiskValues.UNHEALTHY
```
Comment (harshilbits) [inline src/modules/85-cv/pages/slos/components/SLOCardSelect/SLOCardSelect.tsx:25]: Need to implement the logic here in next PR.

File: src/modules/85-cv/pages/slos/CVSLOsListingPage.Utils.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import css from './CVSLOsListingPage.module.scss'
+
+export const RenderAlerts: Renderer<CellProps<any>> = ({ row }) => {
+  const slo = row?.original
+  const { noOfActiveAlerts = 0 } = slo || {}
+  return (
+    <Text
+      className={css.titleInSloTable}
+      title={`${noOfActiveAlerts}`}
+      font={{ align: 'left', size: 'normal', weight: 'semi-bold' }}
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/slos/CVSLOsListingPage.Utils.tsx:24]: font={{ align: 'left', size: 'normal', weight: 'semi-bold' }} is used multiple time , lets move to a class

**Review by vasuki-kalluri — APPROVED** [review]

**Comment by harshilbits** [comment]
retrigger jest

**Comment by harshilbits** [comment]
retrigger jest

**Review by devinder-harness — APPROVED** [review]

File: src/modules/85-cv/pages/slos/CVSLOsListingPage.Utils.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/slos/CVSLOsListingPage.Utils.tsx]: Sure , will be moving in the next PR.

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**